stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -44,16 +45,17 @@
 
     private ImageView mCardIconImageView;
     private CardNumberEditText mCardNumberEditText;
+    private CustomWidthSetter mCustomWidthSetter;
     private StripeEditText mCvcNumberEditText;
     private ExpiryDateEditText mExpiryDateEditText;
     private LockableHorizontalScrollView mScrollView;
     private View mCardNumberSpace;
     private boolean mCardNumberIsViewed;
     private @ColorInt int mErrorColorInt;
-    private boolean mIsAmex;
+    private boolean mIsAmEx;
```

Comment:
It was intentional. I've been meaning to use AmEx all along, but I realized I made a mistake a bunch of times.